src/App.js

```diff
@@ -21,6 +29,19 @@ function App() {
         </a>
         <Button variant="contained"> Hello world!</Button>
         <NavBar></NavBar>
```

Comment:
everything above should really be its own component,  Can you let me know what the purpose is of all the stuff above? or do you plan to delete it?

Reply:
App component shouldn't have jsx in it, You should really be calling components at the top level components.

Reply:
I was planning to delete it